app/views/content/subjects/maths/_article.html.erb

```diff
@@ -1,189 +1,133 @@
 <div class="row inset">
-  <section class="col col-720 col-space-m statement">
-    <div>
-      <span class="pink xlarge bold">Maths is&hellip;</span>
-    </div>
-
-    <div>
-      <span class="pink medium bold">
-        <ul class="slashed-list">
-          <li>going into space</li>
-          <li>getting a mortgage</li>
-          <li>predicting the future</li>
-        </ul>
-      </span>
-    </div>
-  </section>
-
   <section class="col col-720">
-    <h2 class="heading-s">The language of everything</h2>
-
     <p>
-      Maths is the language we use to find patterns and understand the world
-      around us. It has provided the solutions to some of history's most
-      intriguing problems.
+      If maths is your passion, you know that it's not just learning formulas or solving equations. It's a way of thinking and problem solving that’s important in every walk of life.
     </p>
 
     <p>
-      Maths is essential to everyday life, critical to science, technology and
-      engineering as well as being fundamental for most jobs.
+      An exciting career spent passing your passion on to the next generation could be for you.
```

Comment:
We say passion twice in quick succession - is that intentional or does it need tweaking?